jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -364,8 +386,17 @@ def _parse_spectrum1d_3d(app, file_obj, data_label=None,
             warnings.filterwarnings(
                 'ignore', message='Input WCS indicates that the spectral axis is not last',
                 category=UserWarning)
+            meta = standardize_metadata(file_obj.meta)
+
+            # store original WCS in metadata. this is a hacky workaround for
+            # converting subsets to sky regions, where the parent data of the
+            # subset might have dropped spatial WCS info
+            meta['_orig_spatial_wcs'] = None
+            if hasattr(file_obj, 'wcs'):
+                meta['_orig_spatial_wcs'] = _get_celestial_wcs(file_obj.wcs)
+
             s1d = Spectrum1D(flux=flux, wcs=file_obj.wcs,
-                             meta=standardize_metadata(file_obj.meta))
+                             meta=standardize_metadata(meta))
```

Comment:
Didn't you already call `standardize_metadata` on L389?

```suggestion
                             meta=meta)
```